example/Gemfile

```diff
@@ -1,6 +1,7 @@
 source 'https://rubygems.org'
 
 # You may use http://rbenv.org/ or https://rvm.io/ to install and use this version
-ruby '2.7.5'
+ruby '>= 2.6.10'
```

Comment:
If the ruby version needed by RN `0.70.14` is greater than or equal to `2.6.10`, why do not leave the actual `2.7.5` version?

Reply:
Yep Copy/Paste error. Good catch 🌈 Thx. Fixed here https://github.com/pagopa/io-react-native-crypto/pull/27/commits/5ef0dac32617817fda0e612e7b670701ef8727f5